src/test/kotlin/com/exactpro/th2/lwdataprovider/TestSseEventSearchRequest.kt

```diff
@@ -0,0 +1,93 @@
+package com.exactpro.th2.lwdataprovider
+
+import com.exactpro.cradle.TimeRelation
+import com.exactpro.th2.dataprovider.grpc.EventSearchRequest
+import com.exactpro.th2.lwdataprovider.entities.exceptions.InvalidRequestException
+import com.exactpro.th2.lwdataprovider.entities.requests.SseEventSearchRequest
+import com.google.protobuf.Timestamp
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestSseEventSearchRequest {
+    @Test
+    fun testConstructorParam(){
+        // startTimestamp - 1, endTimestamp - 2, searchDirection - AFTER (default)
+        var params = mutableMapOf("startTimestamp" to listOf("1"),
```

Comment:
You don't reassign the value to a variable. So it can be final (`val` in Kotlin)